watershed_workflow/triangulation_with_streams.py

```diff
@@ -0,0 +1,127 @@
+"""Triangulates polygons with stream network representation"""
+import logging
+import collections
+import numpy as np
+import numpy.linalg as la
+from matplotlib import pyplot as plt
+import scipy.spatial
+import meshpy.triangle
+
+import shapely
+from shapely.geometry import LineString, MultiPoint, Point
+from shapely.ops import split
+
+import watershed_workflow.river_tree
+import watershed_workflow.split_hucs
+from watershed_workflow.triangulation import Nodes, NodesEdges
+
+def orient(e):
```

Comment:
isn't this already in the current triangulation?  Import it from there to avoid duplication.